src/Stratis.Bitcoin.IntegrationTests.Common/Runners/StratisBitcoinPowRunner.cs

```diff
@@ -18,9 +19,15 @@ public StratisBitcoinPowRunner(string dataDir)
             this.Network = KnownNetworks.RegTest;
```

Comment:
Can `public StratisBitcoinPowRunner(string dataDir)` also be removed?

Reply:
+1

Reply:
This has been done. The method that consumes it however has 51 usages, so I resorted to defaulting the network in that method - not ideal I know.